config/hbw.default.yml

```diff
@@ -17,3 +17,5 @@ hbw:
   allowed_request_origins:
     - http://*
     - https://*
+
+  filter_bp: false
```

Comment:
it's very difficult to understand what kind of filter we are talking about, need to find more feature descriptive name, e.g.
```
candidate_starters:
  enabled: false
```

Reply:
ok